main.go

```diff
@@ -145,7 +145,7 @@ func establishCustomPath(ctx *cli.Context) error {
 		if currentCtx.IsSet("work-path") && len(providedWorkPath) == 0 {
 			providedWorkPath = currentCtx.String("work-path")
 		}
-		currentCtx = currentCtx.Parent()
+		//currentCtx = currentCtx.Parent() <- FIXME
```

Comment:
@zeripath you added this in #6631 - do you remember what it has done?

on urfave/cli v2 it doesnt exist!

Reply:
I think in v2 there's no need for it. It seems like v1 had the concept of nested contexts, whereas v2 doesn't?

I could definitely be wrong here, though.